src/connection/captcha.js

```diff
@@ -4,22 +4,41 @@ import * as i18n from '../i18n';
 import { swap, updateEntity } from '../store/index';
 import webApi from '../core/web_api';
 
+/**
+ * Return the captcha config object based on the type of flow.
+ * 
+ * @param {Object} m model
+ * @param {Boolean} isPasswordless Whether the captcha is being rendered in a passwordless flow
+ * @param {Boolean} isPasswordReset Whether the captcha is being rendered in a password reset flow
+ */
+export function getCaptchaConfig(m, isPasswordless, isPasswordReset) {
+  if (isPasswordReset) {
+    return l.resetPasswordCaptcha(m);
+  } else if (isPasswordless) {
+    return l.passwordlessCaptcha(m);
+  } else {
+    return l.captcha(m);
+  }
+}
```

Comment:
I'm not sure this is a great API for this function. The two input booleans are actually mutually exclusive when looking at the implementation but someone calling this method would reasonably expect that they can provide any combination of `true` and `false` here and each combination would have its own result.

I can see this pattern used throughout the PR - why was this decision taken vs using something else, maybe a number value and some constants?

Reply:
I just tried to follow the existing pattern with the isPasswordless boolean but you're right it could be confusion to the caller. I'll try to change it to a single value.

Reply:
updated